### PR TITLE
Bp upgrades to filter select

### PR DESCRIPTION
**Pull request by bradparker**
Might want to review this per commit. 
The ESlint stuff came about because _something_ updated and then nothing would run. Ben had the same issue recently with NFP-Profile. Fortunately I could just steal the versions he locked things to in order to get it back up and running.

### CODE REVIEW

**Comment by alexander-bobin** [comment]
LGTM other than whether we need a test for `handleOptionListFocus`

**Comment by bradparker** [comment]
That's a good idea... in fact, theres probably a few more specs which could be added to this.

File: graphs/DeltaArrow/index.js [hunk 1]
```diff
@@ -21,12 +21,12 @@ module.exports = React.createClass({
...
-        loading    = props.loading,
-        emptyState = props.emptyState,
-        showBlankState = (delta === null || delta > 0 || loading === true || emptyState === true),
-        path;
+    var props      = this.props;
+    var delta      = props.delta;
+    var loading    = props.loading;
+    var emptyState = props.emptyState;
+    var showBlankState = (delta === null || delta > 0 || loading === true || emptyState === true);
+    var path;
```
Comment (slwen) [inline graphs/DeltaArrow/index.js]: `vars` be here

File: graphs/DeltaArrow/index.js [hunk 2]
```diff
@@ -42,18 +42,18 @@ module.exports = React.createClass({
...
-          'hui-DeltaArrow--unknown': delta === null && !loading && !emptyState,
-          'hui-DeltaArrow--up': delta > 0 && !loading && !emptyState,
-          'hui-DeltaArrow--down': delta < 0 && !loading && !emptyState
-        });
+    var props      = this.props;
+    var delta      = props.delta;
+    var loading    = props.loading;
+    var emptyState = props.emptyState;
+    var text;
+    var className = classnames({
```
Comment (slwen) [inline graphs/DeltaArrow/index.js]: and more `vars`

File: graphs/LineGraph/LinePath/index.js [hunk 3]
```diff
@@ -79,9 +79,9 @@ module.exports = React.createClass({
   },
 
   calculateTotal: function(dataPoint) {
-    var props = this.props,
-        collectionValueKey = props.collectionValueKey,
-        valueConverter = props.valueConverter;
+    var props = this.props;
+    var collectionValueKey = props.collectionValueKey;
+    var valueConverter = props.valueConverter;
```
Comment (slwen) [inline graphs/LineGraph/LinePath/index.js]: Vars and semi-colons here to be switched out

File: graphs/LineGraph/LinePath/index.js [hunk 4]
```diff
@@ -103,9 +103,9 @@ module.exports = React.createClass({
   },
 
   onMouseOver: function(data, dataPoint, pos) {
-    var props          = this.props,
-        collectionValueKey = props.collectionValueKey,
-        valueConverter = props.valueConverter;
+    var props          = this.props;
+    var collectionValueKey = props.collectionValueKey;
+    var valueConverter = props.valueConverter;
```
Comment (slwen) [inline graphs/LineGraph/LinePath/index.js]: More vars and semi-colons up in hurr

**Comment by alexander-bobin** [comment]
LGTM. Waiting for Sam

**Pull request merged**